### ISSUE TITLE
v0.7.x 503 error 

### ISSUE THREAD

**Issue by tvvocold** [ahwayakchih/openshift-nodebb#5]
history:

```
136  rhc app create node nodejs-0.10
137  rhc add-cartridge http://cartreflect-claytondev.rhcloud.com/reflect?github=smarterclayton/openshift-redis-cart -a node 
138  rhc cartridge add mongodb-2.4 -a node
139  rhc cartridge add mongodb-2.4 -a node 
140  cd node
141  git remote add upstream -m master https://github.com/NodeBB/NodeBB.git
142  git pull --no-edit -s recursive -X theirs upstream v0.7.x
143  git remote add openshift -m master https://github.com/ahwayakchih/openshift-nodebb.git
144  git push origin master
```

url:
http://node-fff2100.rhcloud.com/
BTW, when i restart it i got this info:
Warning: Gear 5577ed41500446abef0000b9 is using 92.1% of disk quota

Maybe we can create another gear to server mongodb , then add another gear to server redis?
:)

**Comment by ahwayakchih** [comment]
Hi,

I've just tested installation process with current v0.7.x branch, it worked ok, except for:
1. i kept getting 503 error for a few minutes after restart: my guess is that some network routes must be refreshed on OpenShift's side.
2. Login page keeps throwing some JS error, but that may be some bug in current NodeBB code.

I noticed that you have added both Redis and MongoDB databases (and MongoDB was added twice?). Maybe that is why you got such error?

I'll update README to make it more clear that only one of the databases is needed.
